src/modm/platform/usb/stm32/usb.hpp.in

```diff
@@ -28,7 +29,7 @@ public:
 	static void
 	initialize(uint8_t priority=3)
 	{
-		static_assert(SystemClock::Usb == 48_MHz, "{{ name }} must have a 48MHz clock!");
+		static_assert(isValueInTolerance(48_MHz, SystemClock::Usb, 0.1_pct), "{{ name }} must have a 48MHz clock!");
```

Comment:
When running the F411-Disco @96Mhz with `.pllM == 7`, SystemClock::Usb becomes 47.999994 Mhz instead of clean 48 Mhz.

Althought `ExternalCrystal / pllM * pllN / pllQ = 8 Mhz / 7 * 336 / 8 = 48 Mhz`, i think 47.999994 is physically correct so i've added some tiny tolerance here.